tools/cli/commands.js

```diff
@@ -406,6 +408,16 @@ function doRunCommand(options) {
     runLog.setRawLogs(true);
   }
 
+  //Opens a browser window when it finishes building
+  if (options.open) {
+    if(process.env.ROOT_URL){
+      open(process.env.ROOT_URL)
+    } else {
+      open('https://localhost:3000')
```

Comment:
I think the port should be taken from the `port` option and default to 3000 only if none is defined.

Reply:
Updated it, if a port is provided it will use it, if not it should default to 3000